src/utils.ts

```diff
@@ -79,9 +78,11 @@ export const getFontFamilyString = ({
 }: {
   fontFamily: FontFamilyValues;
 }) => {
-  for (const [fontFamilyString, id] of Object.entries(FONT_FAMILY)) {
+  for (const [fontFamilyString, { id, fallback }] of Object.entries(
+    FONT_FAMILY,
+  )) {
     if (id === fontFamily) {
-      return `${fontFamilyString}, ${WINDOWS_EMOJI_FALLBACK_FONT}`;
+      return `${fontFamilyString}, ${fallback}`;
```

Comment:
we need to add the `WINDOWS_EMOJI_FALLBACK_FONT` to all fonts. Either in all the separate fallbacks upstream, or here as it was before

Reply:
Ok, I am assuming in the final position at the very end? I will add to all font fallbacks (I think it is already there for one of them)

Oh and why is it needed?

Reply:
I did this in the new PR here:

- #6550